data-collection/src/main/java/refactoringml/App.java

```diff
@@ -142,6 +142,7 @@ public Project run () throws Exception {
 				commitIdToProcess = null;
 
 				// we define a timeout of 20 seconds for RefactoringMiner to find a refactoring.
+				//TODO: make the timeout a config setting, thus we could change it easily e.g. for testing
```

Comment:
Agreed! Maybe you wanna open an issue for this, so that we won't forget it?

Reply:
Done

Reply:
Done #35